routes/file_upload.py

```
@@ -35,3 +35,41 @@
 MIME_TEXT_PLAIN = "text/plain"
 
 
+@router.post("/upload", response_model=FileUploadResponse)
+async def upload_file(
+    file: UploadFile = File(...),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)